src/object/cli_ec.c

```diff
@@ -269,6 +315,7 @@ static void
 ec_free_params(struct ec_params *head)
 {
 	D_FREE(head->iods);
+	/*
```

Comment:
If a block is no longer relevant, best to just remove it rather than comment it out.

Reply:
Actually, this block of code is very relevant and was only disable during some trouble-shooting.  I will re-test with these much-needed lines of code in place.